ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1949,3 +1949,40 @@ periodics:
     - name: build-farm-credentials
       secret:
         secretName: promoted-image-governor
+- agent: kubernetes
+  cluster: build01
+  cron: '@hourly'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  labels:
+    ci.openshift.io/role: infra
```

Comment:
Please remove this (it alerts DPTP on failure). You may consider replacing it with a `reporter_config` to notify on a slack channel.